test/integration/local/conftest.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption('--docker-base-name', default='sagemaker-tensorflow-serving')
+    parser.addoption('--framework-version', default='1.12', choices=['1.11', '1.12'])
```

Comment:
i would change this into --framework-versions and allow this to be a list or a string with versions separated by commas.

Reply:
pytest does accept fixture names as the value of @pytest.mark.parametrize(), so we can't directly parametrize the container fixture in test_container.py